docs/reference/errors-and-warnings/NU5129.md

```diff
@@ -11,9 +11,7 @@ f1_keywords:
 
 # NuGet Warning NU5129
 
-```
-At least one .<extension> file was found in '<build_folder>/<tfm>/', but '<build_folder>/<tfm>/<package_id>.<extension>' was not.
-```
+> At least one .<extension> file was found in '<build_folder>/<tfm>/', but '<build_folder>/<tfm>/<package_id>.<extension>' was not.
```

Comment:
```suggestion
> At least one .&lt;extension&gt; file was found in '&lt;build_folder&gt;/&lt;tfm&gt;/', but '&lt;build_folder&gt;/&lt;tfm&gt;/&lt;package_id&gt;.&lt;extension&gt;' was not.
```

Reply:
Ouch. That's probably something that needs fixed in other errors as well.

Reply:
the docs compiler only warning about this one file, I think.